spec.bs

```diff
@@ -3480,7 +3480,10 @@ Each {{InterestGroupReportingScriptRunnerGlobalScope}} has a
   1. If [=this=]'s [=relevant global object=]'s [=InterestGroupReportingScriptRunnerGlobalScope/
      reporting beacon map=] is not null, then [=exception/Throw=] a {{TypeError}}.
 
-  1. [=map/For each=] |url| of |map|'s [=map/values=]:
+  1. [=map/For each=] |type| → |url| of |map|'s [=map/values=]:

```

Comment:
of |map|, not |map|'s [=map/values=], since you use both keys and values here.
See https://infra.spec.whatwg.org/#map-iterate.